pkg/controller/kfservice/reconcilers/istio/virtualservice_reconciler.go

```diff
@@ -14,107 +14,101 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
cleaner to name this file virtualservice.go?

Reply:
follows the pattern we've used elsewhere - service_reconciler (and route_reconciler, that this replaced). Also there is already a file called just virtualservice.go in the istio package that this uses - so it seems nice to disambiguate?